### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by parvez-rashid**
Created the login page.

There seems to be a problem in the preview, I've centered the form in code but the preview isn't showing it. View the screenshot of it running on the local machine.

image

### CODE REVIEW

File: login/index.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            name="input-pwd"
+            class="pwd-box"
+            id="input-pwd"
+            placeholder="********"
+          />
+
+          <button class="btn btn-primary" type="submit">Login</button>
+          <span class="text-size-sm"
+            >Don't have an account?
+            <a class="link text-style-underline" href="">Sign up</a></span
```
Comment (malaykhakhar) [inline login/index.html:65]: UI suggestion: You can change the color of sign up text so that it matches with your UI.

**Review by malaykhakhar — APPROVED** [review]
The code looks good. You can merge it.

File: login/index.html (hunk 1 above)
Comment (parvez-rashid) [reply login/index.html:65]: Yes, will try to do this, thanks.

**Review by Sahil25061999 — APPROVED** [review]
Looks good , a suggestion from my side would be to make the login button pink as it appears to be the primary color of your page. Also make the login container centered.

**Comment by parvez-rashid** [comment]
> Looks good , a suggestion from my side would be to make the login button pink as it appears to be the primary color of your page. Also make the login container centered.

I used the primary button here from my CL, didn't change the color as I felt blue went well as an accent to the primary color. And yes the login container is actually centered in the code, the preview seems to be broken, will fix it. Thanks.

**Pull request merged**